practices/graded-browser-support.md

```diff
@@ -0,0 +1,99 @@
+Identified vs. Unknown There are over 10,000 browser brands, versions, and configurations and that number is growing. It is possible to group known browsers together.
+
+Capable vs. Incapable No two browsers have an identical implementation. However, it is possible to group browsers according to their support for most web standards.
+
+Modern vs. Antiquated As newer browser versions are released, the relevancy of earlier versions decreases.
+Common vs. Rare There are thousands of browsers in use, but only a few dozen are widely used.
+
+
+
+## Three Grades of Support
+
+### C-grade
+
+C-grade is the base level of support, providing core content and functionality. It is sometimes called core support. Delivered via nothing more than semantic HTML, the content and experience is highly accessible, unenhanced by decoration or advanced functionality, and forward and backward compatible. Layers of style and behavior are omitted.
+
+C-grade browsers should be identified on a blacklist.
+
+Summary: C-grade browsers are identified, incapable, antiquated and rare.
+
+
+### A-grade
+
+A-grade support is the highest support level. By taking full advantage of the powerful capabilities of modern web standards, the A-grade experience provides advanced functionality and visual fidelity.
+
+A-grade browsers should be identified on a whitelist. Approximately 96% of our audience enjoys an A-grade experience.
+
+Summary: A-grade browsers are identified, capable, modern and common.
+
+
+### X-grade
+
+X-grade provides support for unknown, fringe or rare browsers as well as browsers on which development has ceased. Browsers receiving X-grade support are assumed to be capable. (If a browser is shown to be incapable - if it chokes on modern methodologies and its user would be better served without decoration or functionality - then it should considered a C-grade browser.)
+
+X-grade browsers are all browsers not designated as any other grade.
+
+Summary: X-grade browsers are assumed to be capable and modern.
+
+
+### The Relationship Between A-grade and X-grade Support
+
+Unlike the C-grade, which receives only HTML, the X-grade receives everything that A-grade does. Though a brand-new browser might be characterised initially as a X-grade browser, we give its users every chance to have the same experience as A-grade browsers.
+
+## Conclusion
+
+Graded Browser Support provides an inclusive definition of support and a framework for taming the ever-expanding world of browsers and frontend technologies.
+
+Tim Berners-Lee, inventor of the World Wide Web and director of the W3C, has said it best:
+
+> > "Anyone who slaps a 'this page is best viewed with Browser X' label on a Web page appears to be yearning for the bad old days, before the Web, when you had very little chance of reading a document written on another computer, another word processor, or another network."
+
+
+- Reproduced from guidelines by [http://nate.koechley.com/](Nate Koechely)
```

Comment:
Are we citing the source properly? I can't seem to find the original post, but his site has a CC BY 2.5 license, which states:

> Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made.

Any idea where the source is? @dotcode ?